data_safe_haven/utility/logger.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, fmt: str, datefmt: str, *args: Any, **kwargs: Any):
             tracebacks_show_locals=True,
             **kwargs,
         )
+        print("hola chicos")
```

Comment:
:eyes:

Reply:
As you can see from the example comment here, https://github.com/craddm/data-safe-haven/pull/18#issuecomment-2100708446, the report from that bot does indeed provide links to where the coverage has changed and how!

Reply:
That is just an action that works with `pytest-cov`?

That seems easier than making an account on Codecov.

Reply:
Yes, I'm leaning towards this one for exactly that reason. Codecov was easy enough to do via my account, but then we are reliant on them and have some extra - albeit minor - management to do

Reply:
I would vote to go with the action then.